src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const i in actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```
Используй цикл for..of для массивов